dropdownlist.js

```diff
@@ -0,0 +1,10 @@
+const dropDowns = document.querySelectorAll(".dropdown-btn");
+const dropDownContainers = document.querySelectorAll(".dropdown-container");
+console.log(dropDownContainers);
+
+for (let i = 0; i < dropDowns.length; i++) {
+  dropDowns[i].onclick = (e) => {
+    e.target.classList.toggle("active");
+    dropDownContainers[i].classList.toggle("active");
```

Comment:
Em có thể thử từ khóa `this` ở đây nhé.
Có thể sử dụng thằng `nextElementSibling`, không cần khai báo thêm biến dropDownContainers

Reply:
Nhớ tập đặt tên class js-*** như chị Trang nói nhé e

Reply:
dạ anh